EdFi.OdsApi.Sdk/Client/ApiClient.cs

```diff
@@ -496,7 +496,7 @@ private ApiResponse<T> Exec<T>(RestRequest req, RequestOptions options, IReadabl
             {
                 try
                 {
-                    response.Data = (T) typeof(T).GetMethod("FromJson").Invoke(null, new object[] { response.Content });
+                    response.Data = (T)typeof(T).GetMethod("FromJson").Invoke(null, new object[] { response.Content });
```

Comment:
I ran `dotnet format` on the entire project, hence why you'll see a lot of little spacing fixes.